server/mergin/auth/forms.py

```diff
@@ -64,6 +65,11 @@ class RegisterUserForm(FlaskForm):
     def validate(self):
         if not super().validate():
             return False
+        if not is_email_address_valid(self.email.data):
```

Comment:
Is this the only form, what about email update? Maybe we can create custom email validator?